test/Microsoft.Framework.Localization.Test/ResourceManagerStringLocalizerTest.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Resources;
+using Moq;
```

Comment:
See https://github.com/Moq/moq4/issues/168 and https://github.com/castleproject/Core/issues/90

Still coming